### ISSUE TITLE
INVITY Missing directional information for user in Trade

### ISSUE THREAD

**Issue by wendys-cats** [trezor/trezor-suite#2675]
OS: NixOS
Browser: Chromium , version 85
Wallet Version: web, commit 2f07979
Firmware Version: TT 2.3.4 , revision 50854b921 (public)

**Describe the bug**
When opening Suite Coinmarket, directional information for user is missing - amount resctriction and format. 
amount it restricted to 30Eur or higher and format depends on coin (at most two decimal points for FIAT, 8 decimal points for BTC, 6 decimal points for XRP, etc). 
Directional information about amount restriction is vital to the user and user should not be expected to find out this piece of information through experiments, especially when it is a financial matter AND this restriction does not apply as "general knowledge" (f.e. entering letters into this field instead of numbers). 

**Steps to reproduce:**
1.Open Suite Coinmarket (Accounts > Trade)
2. Enter an invalid input (amount lower than 30Eur)

**Expected result:**
Expected to have this information displayed right away.

**Actual result:**
User will be given a notice about the amount restriction only after failing this validation.

**Comment by tsusanka** [comment]
@wendys-cats I think this is a duplicate of #2668?

**Comment by wendys-cats** [comment]
@tsusanka not a duplicate. This issue specifies missing directional information for user such as price limit - usage information that should be displayed for the user. It does not address functionality of the feature nor faulty validation that #2668 is about.

**Comment by tsusanka** [comment]
I am not sure I follow. So you are missing some info dialog where the limits are written? Could you elaborate on some example maybe?

I don't think #2668 is about faulty validation. It is about the fact that when the minimum is inserted to the input the error message is not displayed right away but only after clicking on the "Compare offers".

**Comment by wendys-cats** [comment]
Yes, missing info about price limit before entering anything. 
The problem is that user does not know that there is a 30Eur minimum amount requirement. Since it is not an intuitive knowledge (like, f.e. not putting letters/special characters into field for FIAT/coin amounts) and it pertains to how the Suite Coinmarket works, it should be displayed somewhere (maybe via tooltip? placeholder?), so user can access this information by default, and not only after putting in incorrect amounts. It will improve user experience. 

A correct validation would be consistent about when it informs the user of an input error, hence the validation is deemed faulty.

**Comment by martinboehm** [comment]
We do not have this information (min-max range) before we request offers from our partners. 

The limits are not simple - they depend on fiat/crypto pair, country from which the user is accessing the site, current market situation, if the user is recognized by the partners and even the time of the day (strangely enough some products, for example using bank transfers, are offered only during European business hours).

Any up front information could be misleading. We would have to do a fake request to our partners before the user enters the the form, which has its own problems - spamming partners with unnecessary requests plus the requests take some time, delaying loading of the form.

However, we notice your feedback and will think about some solution how to improve the user experience, maybe caching the ranges from previous requests and displaying somewhere info like "Approximate amount range is 12 USD - 3212 USD" but it will require some analysis, how to do it efficiently.

**Comment by wendys-cats** [comment]
@martinboehm thank you for taking the time to explain, it now makes sense why we do nto display the info.
